homeassistant/components/matter/climate.py

```diff
@@ -448,4 +530,19 @@ def _get_temperature_in_degrees(
         ),
         device_type=(device_types.Thermostat, device_types.RoomAirConditioner),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.CLIMATE,
+        entity_description=ClimateEntityDescription(
+            key="MatterTemperatureControl",
+            name=None,
+        ),
+        entity_class=MatterTemperatureControlClimate,
+        required_attributes=(
+            clusters.TemperatureControl.Attributes.FeatureMap,
```

Comment:
Please list the primary attribute first, which is not the featuremap but the setpoint